kotlinx-coroutines-debug/build.gradle

```diff
@@ -21,7 +21,11 @@ configurations {
 dependencies {
     compileOnly "junit:junit:$junit_version"
     shadowDeps "net.bytebuddy:byte-buddy:$byte_buddy_version"
-    shadowDeps "net.bytebuddy:byte-buddy-agent:$byte_buddy_version"
+    compile "net.bytebuddy:byte-buddy-agent:$byte_buddy_version"
```

Comment:
After this change, any attempt to use `kotlinx-coroutines-debug` as `-javaagent` fails with:
```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:386)
	at sun.instrument.InstrumentationImpl.loadClassAndCallPremain(InstrumentationImpl.java:401)
Caused by: java.lang.NoClassDefFoundError: kotlinx/coroutines/repackaged/net/bytebuddy/agent/Installer
	at kotlinx.coroutines.debug.AgentPremain.premain(AgentPremain.kt:16)
	... 6 more
Caused by: java.lang.ClassNotFoundException: kotlinx.coroutines.repackaged.net.bytebuddy.agent.Installer
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:338)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```

Please fix that.
Also, this is a really worthy feature to test. I would suggest adding the corresponding test with `-javaagent` to `publication-validator` and rename it (the module) to something like `integration-test`

Reply:
Added the test.